### PR TITLE
Add doc files from pyiron

### PR DESCRIPTION
**Pull request by pmrv**
I cut the stuff that is already part of [this](https://github.com/pyiron/pyiron_base/pull/105), except for the some basic intro + installation.  Let me know if you think there's too much duplication somewhere.

I left all example etc. still there, but it might be worthwhile to collect all high-level stuff over on pyiron and only have API docs here.  I have no strong opinions on this though.

### CODE REVIEW

**Comment by pmrv** [comment]
If no one has comments after the meeting tomorrow I'll merge this as is.

**Comment by samwaseda** [comment]
Sorry I'm too horrible in this kind of stuff. I de-assign myself :D

**Comment by pmrv** [comment]
Btw. link to the latest generated build is [here](https://pyiron-atomistics.readthedocs.io/en/docs/).

**Comment by niklassiemer** [comment]
Thanks for the link again, so at least I was looking at the right place :)

File: docs/source/software.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+OpenKIM
+=======
+OpenKIM is a cyberinfrastructure for improving the reliability of molecular and multiscale simulations of materials. It includes a repository of interatomic potentials that are exhaustively tested, tools to help select among existing potentials and develop new ones, and standard integration methods for using potentials in major simulation codes. Visit the `OpenKIM Website
+<https://openkim.org/>`_.
+
+OVITO
+=====
+OVITO is a scientific visualization and analysis software for atomistic and particle simulation data. It helps scientists gain better insights into materials phenomena and physical processes. The program is freely available for all major platforms under an open source license. It has served in a growing number of computational simulation studies as a powerful tool to analyze, understand and illustrate simulation results. https://www.ovito.org
```
Comment (niklassiemer) [inline docs/source/software.rst]: I think we should remove this, shouldn't we?
However, we are not using this file right now, so we should remove Ovito from our pyiron.org page and probably also this entire file.

File: docs/source/about.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+• Object oriented job management – for scaling complex simulation protocols from single jobs to high-throughput simulations.
+
+pyiron (called pyron) is developed in the `Computational Materials Design department <https://www.mpie.de/CM>`_ of `Joerg Neugebauer <https://www.mpie.de/person/43010/2763386>`_ at the `Max Planck Institut für Eisenforschung (Max Planck Institute for iron research) <https://www.mpie.de/2281/en>`_. While its original focus was to provide a framework to develop and run complex simulation protocols as needed for ab initio thermodynamics it quickly evolved into a versatile tool to manage a wide variety of simulation tasks. In 2016 the `Interdisciplinary Centre for Advanced Materials Simulation (ICAMS) <http://www.icams.de>`_ joined the development of the framework with a specific focus on high throughput applications. In 2018 pyiron was released as open-source project.
+
+************
+Getting Help
+************
+Technical issues and bugs should be reported on `Github <https://github.com/pyiron>`_ all other questions can be asked on `stackoverflow using the tag pyiron <https://stackoverflow.com/questions/tagged/pyiron>`_. 
+
```
Comment (niklassiemer) [inline docs/source/about.rst]: The release history is 
1. ancient 
2. for pyiron not pyiron_atomistics, thus versioning is wrong
So let's remove this or state something like 
```
Release 0.2.0 (2021)
=============
* Split pyiron_atomistics from pyiron 0.3.23 (see pyiron release history for previous development)
```

File: docs/source/hdf5.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
+===============================
```
Comment (niklassiemer) [inline docs/source/hdf5.rst]: Do we want this `pyiron_base` developer information at this place at all?

File: docs/source/citation.rst [hunk 4]
```diff
@@ -0,0 +1,123 @@
+.. _citation:
+
+======
+Citing
+======
+The pyiron integrated development environment (IDE) for computational materials science - pyiron IDE - is based on a flexible plugin infrastructure. So depending on which modules are used please cite the corresponding papers.
+
+*****************************
+pyiron paper (accepted)
```
Comment (niklassiemer) [inline docs/source/citation.rst]: Here, we could switch to 'published' or rather just leave it out.

File: docs/source/developers.rst [hunk 5]
```diff
@@ -0,0 +1,5 @@
+.. _developers:
+.. include:: ../../CONTRIBUTING.rst
```
Comment (niklassiemer) [inline docs/source/developers.rst]: I did not find this file in this PR, however, there is a sentence 
'as ASAP as possible' which is weird.

File: docs/source/software.rst (hunk 1 above)
Comment (pmrv) [reply docs/source/software.rst]: Yes, I'll throw the whole file out.  Having it on pyiron.org should be enough.  Having Ovito there or not sounds like a political issue for later.

File: docs/source/about.rst (hunk 2 above)
Comment (pmrv) [reply docs/source/about.rst]: I threw it out for now.  When we start tracking a new changelog we can link it from there.

File: docs/source/citation.rst (hunk 4 above)
Comment (pmrv) [reply docs/source/citation.rst]: Citing links to the original repo, so I'll throw the whole file.

File: docs/source/developers.rst (hunk 5 above)
Comment (pmrv) [reply docs/source/developers.rst]: Grepped all three repo, but can't find it.  Which page did you see it on?

Comment (pmrv) [reply docs/source/developers.rst]: Ah, got it.

File: docs/source/hdf5.rst (hunk 3 above)
Comment (pmrv) [reply docs/source/hdf5.rst]: Nope, axed it.

**Comment by pmrv** [comment]
Thanks for taking the time @niklassiemer!

**Pull request merged**